### ISSUE TITLE
How to reproduce the effect in the paper

### ISSUE THREAD

**Issue by Haiboku233** [daitao/SAN#20]
Hi，Dai Tao.
I ran the code in TrainSAN_script.sh and readme(both for scale=4) to train the model,but it just achieved a PSNR about 31.67 on SET5(it dosen't rise after about 700epoch). What's wrong with that, or what did I missed？
Thanks.

**Comment by Missdonghui** [comment]
Hi, daitao,thanks for your work about SAN.
 But I have some questions, according to the parameters of your paper:"   In each min-batch, 8 LR color patches with size 48 ¡Á 48 are provided as inputs. Our model is trained by ADAM optimizor with ¦Â1 = 0.9, ¦Â2 = 0.99, and ¦Å = 106Ó18 . The learning rate is initialized as 106Ó14 and then reduced to half every 200 epochs. Our proposed SAN has been implemented on the 
 Pytorch framework [23] on an Nvidia 1080Ti GPU. "The difference is that I use Nvidia 1080 GPU. I cannot reproduce the PSNR=38.31(ste5 scale=2) even use the best model,only  achieved 37.8336.Can you give me  more details about it ,thanks a lot . Looking forward to you

**Comment by Awenjie10** [comment]
> Hi, daitao,thanks for your work about SAN.
> But I have some questions, according to the parameters of your paper:" In each min-batch, 8 LR color patches with size 48 ¡Á 48 are provided as inputs. Our model is trained by ADAM optimizor with ¦Â1 = 0.9, ¦Â2 = 0.99, and ¦Å = 10�6Ó18 . The learning rate is initialized as 10�6Ó14 and then reduced to half every 200 epochs. Our proposed SAN has been implemented on the
> Pytorch framework [23] on an Nvidia 1080Ti GPU. "The difference is that I use Nvidia 1080 GPU. I cannot reproduce the PSNR=38.31(ste5 scale=2) even use the best model,only achieved 37.8336.Can you give me more details about it ,thanks a lot . Looking forward to you

hi, have you solved the gap between yours and the paper's? I met the same question. @daitao
